src/main/java/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,163 @@
+package internet.shop.dao.jdbc;
+
+import internet.shop.dao.OrderDao;
+import internet.shop.exceptions.DataProcessingException;
+import internet.shop.lib.Dao;
+import internet.shop.model.Order;
+import internet.shop.model.Product;
+import internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long id) {
+        List<Order> orders = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(
+                     "SELECT * FROM orders WHERE user_id = ?;");
```

Comment:
Maybe you should check if `deleted = false` everywhere where you want to get an item